### PR TITLE
[MAISTRA-664] 1.0.0 docs updates

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: topics/docs/examples/bookinfo.adoc [hunk 1]
```diff
@@ -18,26 +18,18 @@ Install Maistra by following the instructions link:/docs/getting_started/install
...
 
-=== Update Security Context Constraints
-To add the service account used by Bookinfo application to anyuid and privileged SCCs in "bookinfo" project:
-
-```
-  $ oc adm policy add-scc-to-user anyuid -z default -n bookinfo
-  $ oc adm policy add-scc-to-user privileged -z default -n bookinfo
-```
-
 === Deploy Bookinfo Application
```
Comment (jwendell) [inline topics/docs/examples/bookinfo.adoc]: Probably keep the privileged one, noting that it's still required for OCP 3 (where CNI is not used)?

File: topics/docs/getting_started/install.adoc [hunk 2]
```diff
@@ -154,7 +150,7 @@ Maistra supports a shorthand of *smcp* for *ServiceMeshControlPlane* and *smmr*
 ==== Single-Tenant Installation
 
 
-Single tenant mode is disabled by default in TP12. To enable it, set multitenant to false in your *ServiceMeshControlPlane*. In order to deploy the Istio Control Plane, we need to create a *ServiceMeshControlPlane* such as the one in the following example. In our examples, we use the istio-system project. For more information on the parameters and their configuration please see the link:../custom-install[custom installation documentation].
+Single tenant mode is disabled by default. To enable it, set multitenant to false in your *ServiceMeshControlPlane*. In order to deploy the Istio Control Plane, we need to create a *ServiceMeshControlPlane* such as the one in the following example. In our examples, we use the istio-system project. For more information on the parameters and their configuration please see the link:../custom-install[custom installation documentation].
```
Comment (jwendell) [inline topics/docs/getting_started/install.adoc:154]: Single tenant mode is not supported at all, it's not just disabled by default.

File: topics/docs/getting_started/install.adoc [hunk 3]
```diff
@@ -166,7 +162,7 @@ oc create -n istio-system -f <name of file>
 ```
 
 ==== Multi-Tenant Installation
-Multitenancy is enabled by default in TP12. To use it, create a project to contain the control plane, and create the *ServiceMeshControlPlane* in that project.
+Multitenancy is enabled by default. To use it, create a project to contain the control plane, and create the *ServiceMeshControlPlane* in that project.
```
Comment (jwendell) [inline topics/docs/getting_started/install.adoc:166]: Multitenancy is the only way to go. There will be no setting to enable/disable it.

File: topics/docs/getting_started/install.adoc (hunk 2 above)
Comment (brian-avery) [reply topics/docs/getting_started/install.adoc:154]: Single tenancy is still supported, I think the docs changes for that should be made when that functionality is removed.

File: topics/docs/examples/bookinfo.adoc (hunk 1 above)
Comment (brian-avery) [reply topics/docs/examples/bookinfo.adoc]: Added back in. Thanks.

**Review by jwendell — APPROVED** [review]

**Review by dmitri-d — APPROVED** [review]
lgtm

**Pull request merged**